APIs.js

```diff
@@ -0,0 +1,4 @@
+export const LoginEndpoint = '/users/login'
+export const SignupEndpoint = '/users'
+export const TokenFile = 'token'
+export const UserFile = 'user'
```

Comment:
Please add new empty line at the end of each file, you can also configure your editor to do that automatically.